ports/nrf/modules/manifest.py

```diff
@@ -1,2 +1,3 @@
-module("_mkfs.py", base_path="$(PORT_DIR)/modules/scripts", opt=3)
+# module("_mkfs.py", base_path="$(PORT_DIR)/modules/scripts", opt=3)
+freeze("$(PORT_DIR)/modules/scripts")
```

Comment:
It's not clear why this change is needed, what was the reason?  I think just revert it.  If a board needs more scripts then it should specify them in its own `manifest.py`.

Reply:
It's more a test thing, so I can add files to be frozen which do not go into the main line.

Reply:
Reverting it just requires to drop that last commit. Another question about generated files: The XIAO board firmware can be loaded as .uf2 file, which is convenient for users. For the Nordic Semi port, usually the .bin and .hex files are created. So I added instructions in deploy.md to create the .uf2 file.  Is is possible to add .uf2 file creation to a single or few boards as well? I did not check whether other boards support .uf2 file loading as well. The Arduino Nano Connect board could be a candidate.

Reply:
Reverted.

Reply:
> is it possible to add .uf2 file creation to a single or few boards as well?

Yes we can do that, like it's done on samd.

Reply:
At the SAMD port the .uf2 file is created for all boards. But here it would only be needed for the SEEED_XIAO_NRF52 board.